src/figuresClasses.ts

```diff
@@ -1,11 +1,83 @@
-export interface Figure {}
+import { error } from 'console';
 
-export class Triangle implements Figure {}
+export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  constructor(
+    public shape: 'triangle',
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
```

Comment:
[CODE KNOWLEDGE] - You don't need to manually assign constructor parameters to class properties when using the access modifier 'public'. TypeScript does this automatically.